assembly/massa/message_trigger/main.ts

```diff
@@ -0,0 +1,14 @@
+import { generateEvent, createSC, call, Address, Args, getOpData, toBytes } from "@massalabs/massa-as-sdk";
+
+function createContract(): Address {
+    const bytes: StaticArray<u8> = getOpData(toBytes("smart-contract"));
```

Comment:
Maybe add a small comment with the path of the retrieved smart contract?

Reply:
Added.